script/fetch_content.py

```diff
@@ -4,27 +4,30 @@
 import re
 
 
-def arguments():
+def fetch_arguments():
   parser = argparse.ArgumentParser()
   parser.add_argument('--url', help='add url')
   parser.add_argument('--text', help='add text')
-  args = parser.parse_args()
+  args = parser.parse_args() 
   return args
-def add_characters():
+
+
+def print_10_characters_before_and_after_text():
```

Comment:
Please add documentation (doctoring) to this function.
Describe what it returns.